docs/configure_admin_workstation_post_install.rst

```diff
@@ -59,10 +59,10 @@ beginning of every session, and sets up SSH host aliases for the servers.
 The only thing you need to remember to do is enable
 persistence when you boot the *Admin Workstation*. If you are
 using the *Admin Workstation* and are unable to connect to any
-of the authenticated Onion Services, restart Tails and make
+of the authenticated onion services, restart Tails and make
 sure to enable persistence.
 
-.. _Authenticated Tor Onion Services: https://www.torproject.org/docs/tor-manual.html.en#HiddenServiceAuthorizeClient
+.. _authenticated onion services: https://tb-manual.torproject.org/onion-services/#onion-service-authentication
```

Comment:
This links to the docs for in-browser authentication, which sadly does not work under tails. This might be a better one given the context: https://community.torproject.org/onion-services/advanced/client-auth/

Reply:
:heavy_check_mark: Done in 54af89906dd20db0b41053b9c8dc765750c77827

Reply:
:heavy_check_mark: